megazords/cirrus/tests/python-tests/resources/test.fml.yml

```diff
@@ -28,6 +28,7 @@ features:
         value: {
            "something": "wicked"
         }
+    allow-coenrollment: false
```

Comment:
You can probably delete this line now that `allow-coenrollment: false` is the default.